eng/test-job.yml

```diff
@@ -131,7 +133,7 @@ jobs:
     - template: /eng/send-to-helix-step.yml
       parameters:
         displayName: Send tests to Helix
-        buildConfig: ${{ parameters.buildConfig }}
+        buildConfig: $(buildConfigUpper)
```

Comment:
Just out of curiosity, why this is needed?

Reply:
The buildConfig is used to build some paths that are case sensitive.  They need "Release" instead of "release" (parameters.buildConfig).